lib/emque/consuming/adapters/rabbit_mq/manager.rb

```diff
@@ -74,6 +90,18 @@ def initialize_workers
             }
           end
 
+          def initialize_delayed_message_workers
+            self.delayed_message_workers = [].tap { |workers|
```

Comment:
I went with tap + times to follow the same pattern as `initialize_workers` (https://github.com/emque/emque-consuming/pull/55/files/307386f4716e23545d459a60d07f6063fbedd293#diff-4d95204b90cdb755a99c5ae519d36cf2R82)